src/main/java/com/programmers/java/Calculator.java

```diff
@@ -0,0 +1,99 @@
+package com.programmers.java;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Stack;
+
+public class Calculator {
+    static int index;
+    static ArrayList<String> history = new ArrayList<>();
+    public static BigDecimal calculate(String expression) throws Exception {
+        index = 0;
+        String postExp = toPostfix(expression);
+        index = 0;
+        BigDecimal answer = postfixCalculator(postExp);
+        history.add(expression+" = "+answer);

```

Comment:
String.format 문법을 사용해보아요
```suggestion
        history.add(String.format("%s = %s", expression, answer));

```

Reply:
가독성과 유연함을 위해서라도 String.format()을 사용해보겠습니다!